cpp/include/raft/util/cudart_utils.hpp

```diff
@@ -25,6 +25,7 @@
 #pragma once
 
 #include <raft/core/error.hpp>
+#include <raft/core/logger.hpp>
```

Comment:
whats the logger include for ?

Reply:
Good catch. I was logging some things initially but decided to remove it from the function.